### DISCUSSION TITLE
Add button to reset addons

### DISCUSSION THREAD

**Discussion by randewd** [ScratchAddons/ScratchAddons#5663] [Ideas] [answered]
I want to reset the addons, but I can't seem to find a button for it.

**Comment by Samq64** [comment, marked as answer]
See #4754
A workaround is to run `chrome.storage.sync.clear()` in the browser console.

**Comment by DNin01** [comment]
https://github.com/ScratchAddons/ScratchAddons/issues/4754, the suggestion for this feature, was closed as not planned, so this probably won't become a feature for now, except to help [fix critical problems](https://github.com/ScratchAddons/ScratchAddons/issues/5306).

As others have said, opening the browser console with <kbd>ctrl</kbd><kbd>shift</kbd><kbd>j</kbd> and then typing `chrome.storage.sync.clear()` will delete all of the extension's data.

**Reply by randewd** [reply]
But in which tab should I run it?

**Reply by DNin01** [reply]
I don't have much experience with these commands, but I'm guessing it's probably the extension's settings page. If that doesn't work, then it's probably the background page, which you can inspect by turning on developer mode from the extensions list and then there will be some sort of link to it.

**Comment by randewd** [comment]
I've discovered that re-installing the extension also works.
